aspen/http/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+:mod:`http`
+===========
+
+Aspen doesn't implement all of HTTP, only things related to filesystem
+dispatch.
+
+.. automodule:: aspen.http.mapping
+.. automodule:: aspen.http.request
+.. automodule:: aspen.http.resource
+
+"""
```

Comment:
Maybe we should kill the `http` module altogether. We could move `aspen.http.mapping` and `aspen.http.request` into `aspen.mappings`. `aspen.http.resource` could go into the existing `aspen.resources` module.